inspirehep/modules/theme/jinja2filters.py

```diff
@@ -488,6 +488,9 @@ def publication_info(record):
     pub_infos = []
     if 'publication_info' in record:
         for pub_info in record.publication_information:
+            if pub_info.get('hidden'):
+                continue
+
```

Comment:
This looks quite incomplete, as the `publication_info` is used for much more than just displaying it.